README.md

```diff
@@ -13,6 +13,16 @@ This repository contains the [source code](./src) and the [content](./content) f
 
 Read through the [contributing guide](CONTRIBUTING.md) to learn how you can contribute to the Prisma documentation.
 
+## Prerequisites for macOS Monterey (Apple M1 chip)
+
+1. Install [homebrew](https://brew.sh/).
+2. Install libvips via homebrew.
```

Comment:
To make it clear that the command applies to the second pre-req, let's add a phrase like "Install livips via homebrew, using the following command:"

We have some decisions to make and add to our Style Guide... for example, I would bold *livips* to make it stand out as a product, others might use courier font, you left it normal fint (which reads fine), etc. No rush, but eventually let's decide and add to Style Guide.

Reply:
How about nesting the codeblock? A single indentation (tab) didn't work, but a double-indentation nests the codeblock under the second step, so they look connected.

It makes sense to format product names somehow. One consideration that comes to mind with bold is that it is mostly used for UI elements (buttons, checkbox names, tab names, etc.).

Reply:
Joel posted [this](https://github.com/prisma/prisma/blob/main/CONTRIBUTING.md#general-prerequisites) as an example, so adding package names as code snippets with a link to their project seems like a convention we could use.

Reply:
Yep I like how Joël added a link to their projects... and I agree we can use code font if it is a true package name, nice solution. One day we will add that to the Style Guide. ;-)